src/db/users.js

```diff
@@ -0,0 +1,32 @@
+var records = [
+  {
+    id: 1,
```

Comment:
Why start at 1? Find by id is easier if we start at 0.

Reply:
For sure